mix.exs

```diff
@@ -47,12 +47,13 @@ defmodule ConfigCat.MixProject do
 
   defp deps do
     [
-      {:elixir_uuid, "~> 1.2"},
-      {:httpoison, "~> 1.7"},
-      {:jason, "~> 1.2"},
+      {:credo, "~> 1.7", only: [:dev, :test], runtime: false},
```

Comment:
This is the only change to this file; the rest was alphabetizing the dependencies to make things easier to fine.